app.py

```
@@ -1399,6 +1399,66 @@ def generate():
+            domain,
+            depth=3,
+            max_pages=100,
+            max_subdomains=25,
+            baas_validate=True,
+            on_progress=_progress,
+        )
+
+    try:
+        loop = asyncio.get_event_loop()
+        report = await loop.run_in_executor(None, _run)
+    except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)